test/development/acceptance-app/ReactRefreshLogBox-builtins.test.ts

```diff
@@ -1,12 +1,10 @@
 import { createSandbox } from 'development-sandbox'
 import { FileRef, nextTestSetup } from 'e2e-utils'
 import path from 'path'
-import { describeVariants as describe } from 'next-test-utils'
 import { outdent } from 'outdent'
 
-// TODO-APP: Investigate snapshot mismatch
```

Comment:
2 years old. There's a larger task to look at build errors and how we can normalize their display.